apps/omg_watcher/test/omg_watcher/exit_processor/update_db_cores/new_exits_test.exs

```diff
@@ -0,0 +1,142 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
```

Comment:
Module path is `update_db_cores` ❓ Should correspond to the app module, so `update_db`

Reply:
ouch, thanks